cli/fs_util.rs

```diff
@@ -628,4 +670,22 @@ mod tests {
       );
     }
   }
+
+  #[test]
+  fn test_specifier_to_file_path() {
+    run_success_test("file:///", "/");
+    run_success_test("file:///test", "/test");
+    run_success_test("file:///dir/test/test.txt", "/dir/test/test.txt");
+    run_success_test(
+      "file:///dir/test%20test/test.txt",
+      "/dir/test test/test.txt",
```

Comment:
Are URLs like `file://C:/path/to/file` handled correctly?

Reply:
Yeah, the url crate seems to handle them on both windows and unix.